.github/workflows/zz_generated.create_release.yaml

```diff
@@ -0,0 +1,182 @@
+# DO NOT EDIT. Generated with:
+#
+#    devctl gen workflows
```

Comment:
@kopiczko how can I know this is correct now?

Reply:
Have faith. Eventually we should print used devctl version here.